Epam.TestAutomation.Web/PageObjects/Pages/SearchPage.cs

```diff
@@ -0,0 +1,29 @@
+using Epam.TestAutomation.Core.BasePage;
+using Epam.TestAutomation.Core.Browser;
+using OpenQA.Selenium;
+
+namespace Epam.TestAutomation.Web.PageObjects.Pages;
+
+public class SearchPage : BasePage
+{
+    public override string Url => "https://www.epam.com/";
+
+    //first test variables
+    public static string keyWord = "Automation";
+    public string searchUrl = "https://www.epam.com/search?q=Automation";
+    public By searchButton = By.XPath("//span[@class='search-icon dark-iconheader-search__search-icon']");
+    public By searchForm = (By.XPath("//input[@id='new_form_search']"));
+    public By findButton = (By.XPath("//span[@class='bth-text-layer']"));
+
+    public static IEnumerable<IWebElement> listOfArticles =
```

Comment:
Why is is static?

Reply:
so articlesToLower can use it

Reply:
And? No need to make members in this class static. And I don't see that you use static members correct in your framework. So make all members non static.

Reply:
Upper case for all members of the class.